dev-docs/RFCs/v4.0.0/terms_for_concepts_and_datatypes.md

```diff
@@ -0,0 +1,96 @@
+
+Function name changes:
+
+|                  Current name                 |        Proposed name         |
+|-----------------------------------------------|------------------------------|
+| `getH3UnidirectionalEdge`                     | `getDirectedEdge`            |
+| `h3UnidirectionalEdgeIsValid`                 | `isValidDirectedEdge`        |
+| `getOriginH3IndexFromUnidirectionalEdge`      | `getDirectedEdgeOrigin`      |
+| `getDestinationH3IndexFromUnidirectionalEdge` | `getDirectedEdgeDestination` |
+| `getH3IndexesFromUnidirectionalEdge`          | `getDirectedEdgeCells`       |
+| `getH3UnidirectionalEdgesFromHexagon`         | `getDirectedEdgesFromCell`   |
+| `getH3UnidirectionalEdgeBoundary`             | `getDirectedEdgeBoundary`    |
+
+
+## Function Names
+
+A few more function name change proposals:
+
+|          Current name         |     Proposed name     |                  Notes                  |
+|-------------------------------|-----------------------|-----------------------------------------|
+| (doesn't exist)               | `isValidIndex`        |                                         |
+| `h3IsValid`                   | `isValidCell`         |                                         |
+| `h3UnidirectionalEdgeIsValid` | `isValidDirectedEdge` |                                         |
+| `h3IsPentagon`                | `isPentagon`          |                                         |
+| `h3ToParent`                  | `cellToParent`        |                                         |
+| `h3ToChildren`                | `cellToChildren`      |                                         |
+| `h3IndexesAreNeighbors`       | `cellsAreNeighbors`   |                                         |
+| `geoToH3`                     | `geoToCell`           |                                         |
+| `h3ToGeo`                     | `cellToGeo`           |                                         |
+| `kRing`                       | `disk`                | filled-in disk                          |
+| `hexRing`                     | `ring`                | hollow ring                             |
+| `numHexagons`                 | `numCells`            |                                         |
+| `getRes0Indexes`              | `getRes0Cells`        |                                         |
+| `getPentagonIndexes`          | `getPentagons`        |                                         |
+| `h3GetBaseCell`               | `getBaseCellNumber`   |                                         |
+| `h3Distance`                  | `gridDistance`        | no `get` prefix?                        |
+| `h3Line`                      | `cellPath`            | future: `edgePath`, `directedEdgePath`? |
+
+
+For the `kRing`, `hexRange`, `hexRing`, etc. family of functions, should we come up with some standard prefix or suffix to denote that the function will fail if it encounters a pentagon?
```

Comment:
That's only true for `hexRange`, right? Should we even expose that, or just expose `neighbors` and `ring`?

Reply:
This should be reflected in the latest pass.